ui/window.go

```diff
@@ -108,6 +109,15 @@ func CreateWindow(wal *wallet.Wallet, decredIcons map[string]image.Image, collec
 	return win, nil
 }
 
+func (win *Window) refresh() {
+	win.window.Invalidate()
+}
```

Comment:
There's no need to have this as a method if window invalidation is called from the changePage method.